workflow/signals.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+
+from django.contrib.auth.models import Group
+from django.db.models import signals
+from django.dispatch import receiver
+
+from tola import DEMO_BRANCH
+from tola.management.commands.loadinitialdata import (
+    DEFAULT_WORKFLOWLEVEL1_ID, DEFAULT_WORKFLOWLEVEL1_NAME)
+from workflow.models import (TolaUser, WorkflowLevel1, WorkflowTeam,
+                             ROLE_VIEW_ONLY)
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(signals.post_save, sender=TolaUser)
+def add_users_to_default_wflvl1(sender, instance, **kwargs):
```

Comment:
If the TolaUser is being updated instead, you'll create a new WorkflowTeam with the default program, even though the user already has it. You should check this case as well.

Reply:
Good eye! I didn't think about this possibility